.github/workflows/examples.yml

```diff
@@ -6,6 +6,21 @@ on:
   schedule:
     # Run this action daily at 7:00 UTC
     - cron: "0 7 * * *"
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: >
+          [Optional] Repository name with owner. For example, mlflow/mlflow.
+           Defaults to the repository that triggered a workflow.
+        required: false
+        default: ""
+      ref:
+        description: >
+          [Optional] The branch, tag or SHA to checkout. When checking out the repository that
+           triggered a workflow, this defaults to the reference or SHA for that event. Otherwise,
+           uses the default branch.
+        required: false
+        default: ""
```

Comment:
Just adding these inputs actually doesn't do anything. We need to pass them to the `actions/checkout` action.